added new notes tests for ch08 (#17)
cs4560-utah-sp24/test: ch08/ex-get-forms.md

which will either be `GET` or `POST`. Default to using GET when no
attribute is present.

> Ensure not to modify the signature of the URL.request method while 
> implementing `GET` form submissions, where form data is appended to
> the URL query string.

Reviewer: ```suggestion
Do not modify the signature of the `URL.request` method.
Instead, append form data to the URL query string.
```

     <html>
       <body>
         'Mmm'

> Double-check that GET forms are not accompanied by the Content-Length header in the request.

Reviewer: ```suggestion
Double-check that GET forms are not accompanied by the `Content-Length` header in the request.
```
